pkg/actors/actors.go

```diff
@@ -399,7 +399,6 @@ func (a *actorsRuntime) Call(ctx context.Context, req *invokev1.InvokeMethodRequ
 	if lar == nil {
 		lar = &lookupActorRes{}
 	}
-
 	var resp *invokev1.InvokeMethodResponse
```

Comment:
```suggestion
	}

	var resp *invokev1.InvokeMethodResponse
```

Because there'll be a change on this file getting merged very soon and I want to spare a merge conflict :)